service_container/3.3-di-changes.rst

```diff
@@ -567,6 +567,12 @@ Start by updating the service ids to class names:
     classes (e.g. ``Twig_Extensions_Extension_Intl`` instead of ``Twig\Extensions\IntlExtension``),
     you can't redefine the service as ``Twig_Extensions_Extension_Intl: ~`` and
     you must keep the original ``class`` parameter.
+    
+.. caution::
+
+    If a service is processed by `Compiler Pass`_ you could face a  "You have requested a non-existent service" error. 
+    To get rid of this, be sure that Compiler Pass is using ``findDefinition`` instead of ``getDefinition`` as the latter
```

Comment:
I would use `findDefinition()` and `getDefinition()` (i.e. include the parentheses)

Reply:
We could also add a note that it is generally a good idea to check with `has()` first if there actually is an alias or definition we are looking for.